### PR TITLE
Prepare release v3.0.0

### PR DESCRIPTION
**Pull request by ubergesundheit**

### CODE REVIEW

File: helm/kong-app/values.yaml [hunk 1]
```diff
@@ -495,7 +495,7 @@ ingressController:
     # The controller disables TLS verification by default because Kong
     # generates self-signed certificates by default. Set this to false once you
     # have installed CA-signed certificates.
-    kong_admin_tls_skip_verify: "true"
+    kong_admin_tls_skip_verify: true
```
Comment (webwurst) [inline helm/kong-app/values.yaml:498]: Why is this changed? By upstream? Maybe we should still add one line to changelog?

Comment (ubergesundheit) [reply helm/kong-app/values.yaml:498]: Yes, from upstream. Forgot to update the values in previous sync. This is related to a change they reverted themselves in upstream already (That was related to requiring all `env` values to be of type string). So I would not include this in Changelog.

**Review by webwurst — APPROVED** [review]

File: helm/kong-app/values.yaml (hunk 1 above)
Comment (ubergesundheit) [reply helm/kong-app/values.yaml:498]: Introduced by https://github.com/Kong/charts/pull/728
Reverted in https://github.com/Kong/charts/pull/733/files
